### ISSUE TITLE
[v5.5.0] No support for PayPo payments

### ISSUE THREAD

**Issue by wrozwad** [Adyen/adyen-android#1690]
**Describe the bug**
According to the [documentation](https://docs.adyen.com/payment-methods/paypo/android/), we have the possibility to use PayPo payments with this SDK. We should be able to use the `PaymentMethodTypes.PAYPO` value, which unfortunately is not available. Is this payment method supported at all?

**Comment by OscarSpruit** [comment]
Hi @wrozwad, PayPo is actually supported by the SDK. With drop-in it works out of the box and with components you can use the `InstantPaymentComponent` for it. One thing to note is that you have to collect and pass the shopper email and shopper name to the backend yourself as the SDK offers no UI to do this.

**Comment by OscarSpruit** [comment]
@wrozwad yes, it is indeed similar to Google Pay. The only way to avoid this intermediate screen is by integrating `InstantPaymentComponent` directly. This component is a lot easier to integrate with than `GooglePayComponent` though

**Comment by wrozwad** [comment]
@OscarSpruit I decided to continue in this thread. To properly initialize PayPo in test mode, we need to replace the received `PaymentComponentData` before parsing it into JSON, so that we additionally pass test payment data, in accordance with the [documentation](https://docs.adyen.com/payment-methods/paypo/android-component/#test-and-go-live).

```kotlin
fun makePayment(state: PaymentComponentState<*>) { // in this case state is InstantComponentState
    val paymentData = state.data.copy(
        shopperName = ShopperName(firstName = "name", lastName = "surname"),
        shopperEmail = "email@gmail.com",
    )
    ...
}
```

I noticed that in the production version, during the first payment with PayPo, these values are pre-filled in the web form by default. I also discovered a feature outside the documentation that manually adding `telephoneNumber` to `PaymentComponentData` also influences the default filling of the phone number in the form.

However, there is still the address form. I couldn't manage to make the web form fill in the provided values. I tried both `deliveryAddress` and `billingAddress`:
```kotlin
deliveryAddress = Address(
    street = "Test",
    houseNumberOrName = "123",
    postalCode = "00-001",
    city = "Warszawa",
),
```
![image](https://github.com/user-attachments/assets/c175fb62-b47c-4d79-bfdb-a81ca08093b5)
